### PR TITLE
Formatted <code> in email

### PR DESCRIPTION
**Pull request by TedGoas**
Adds support for `code` in email.

### CODE REVIEW

File: docs/email/base/typography.html [hunk 1]
```diff
@@ -122,7 +122,35 @@ <h2 style="font-weight: bold; font-size: 21px; line-height: 24px; color: #242629
     </div>
 </section>
 <section class="stacks-section">
+    {% header h2 | Code %}
+    <div class="stacks-preview mb16">
+{% highlight html linenos %}
+// Preformatted code block
+<pre style="line-height: 17px; background-color: #EFF0F1; padding: 4px 8px; border-radius: 3px; overflow-x: auto;"><code style="margin: 0 0 15px; background-color: #EFF0F1; color: #242729; font-size: 13px; line-height: 17px; font-family: Consolas, Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace, sans-serif;">
```
Comment (ludwiczakpawel) [inline docs/email/base/typography.html]: So `background-color` is specified on both elements: `pre` and `code` but `color` is specified only on `code`. Is that correct? Just making sure this is intended. 

------

Out of curiosity: how does `overflow-x: auto` behave in various email clients?

**Comment by TedGoas** [comment]
@UXPiper _ALL_ of me wishes we stylized `code` with a dark theme 😂 but I'm just matching what we do on the web.

Regarding `target: blank;` I'd prefer not to force folks into that UX (they can launch in a new tab if they want). Sound reasonable?

File: docs/email/base/typography.html [hunk 2]
```diff
@@ -122,7 +122,35 @@ <h2 style="font-weight: bold; font-size: 21px; line-height: 24px; color: #242629
...
+    …
+</code></pre>
 
+// Inline code
```
[the remark this answers is not in the document]
Comment (TedGoas) [reply docs/email/base/typography.html]: @ludwiczakpawel Wow I had no idea this features existed, so cool!

File: docs/email/base/typography.html (hunk 1 above)
Comment (TedGoas) [reply docs/email/base/typography.html]: @ludwiczakpawel [support for `overflow` is _ok_](https://www.campaignmonitor.com/css/positioning-display/overflow/)... but it's supported in the places we need it most: mobile clients and Gmail. [`word-wrap` is an alternative](https://www.campaignmonitor.com/css/text-fonts/word-wrap/).

**Pull request merged**